dhis2-android-app/src/main/java/org/dhis2/mobile/processors/OfflineDataProcessor.java

```diff
@@ -116,6 +123,14 @@ private static void uploadOfflineReports(Context context) {
                     // Removing uploaded data
                     TextFileUtils.removeFile(reportFile);
                     PrefUtils.removeOfflineReportInfo(context, reportFile.getName());
+                } else {
+                    message = message + context.getString(R.string.network_error) + ": "
```

Comment:
@idelcano this code for log mixed with other logic is a little dirty, can you extract a separate method or class?